src/components/WwwFrame/PromotionalBanner/Banners/AppealBanner/AppealBanner.vue

```diff
@@ -1,80 +1,105 @@
 <template>
-	<div class="sitewide-appeal-wrapper">
-		<div class="sitewide-appeal row" @click="toggleAccordion">
```

Comment:
Since this is pretty dialog/modal-esque, adding a listener for the escape key on this div would be nice to let people close this dialog with their keyboard. See line 8 of KvLightbox.vue for an example.

Reply:
✔️